Sources/TuistLoader/Models+ManifestMappers/CodeCoverageMode+ManifestMapper.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+import ProjectDescription
+import TuistGraph
+
+extension CodeCoverageMode {
+    static func from(
```

Comment:
I'd unit test the three scenarios in this method. It's important that we ensure that the right project path is used for the `.targets` mode.